builder/builder-next/adapters/containerimage/pull.go

```diff
@@ -182,7 +182,7 @@ func (is *Source) resolveRemote(ctx context.Context, ref string, platform *ocisp
 		p = *platform
 	}
 	// key is used to synchronize resolutions that can happen in parallel when doing multi-stage.
-	key := "getconfig::" + ref + "::" + platforms.Format(p)
+	key := "getconfig::" + ref + "::" + platforms.FormatAll(p)
```

Comment:
Could use some extra eyes on this one if changing this one is OK to do

Reply:
@tonistiigi: does this PR look OK (in particular the use of `FormatAll()` above).